core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -654,11 +661,14 @@ angular.module('oppia').directive('explorationEditorPage', [
           // Remove save from tutorial if user does not has edit rights for
           // exploration since in that case Save Draft button will not be
           // visible on the create page.
-          if (!GLOBALS.can_edit) {
-            var index = ctrl.EDITOR_TUTORIAL_OPTIONS.indexOf(
-              saveButtonTutorialElement);
-            ctrl.EDITOR_TUTORIAL_OPTIONS.splice(index, 1);
-          }
+          UserExplorationPermissionsService.getPermissionsAsync()
+            .then(function(permissions) {
+              if (!permissions.can_edit) {
+                var index = ctrl.EDITOR_TUTORIAL_OPTIONS.indexOf(
```

Comment:
Make sure that you have tested this behavior.

Reply:
Yes, I have tested this.